base/settings/R/loadPath_sitePFT.R

```diff
@@ -0,0 +1,24 @@
+#' Title loadPath.sitePFT
+#'
+#' @param settings pecan setting list.
+#' @param Path  Character of file name with extension. The path will be generated using the outdir tag in pecan settings.
+#'
+#' @return a dataframe of two columns of site and pft
+#' @export loadPath.sitePFT
+#' @description The csv or the text file needs to have a header and be separated using comma. Under the first column in the text file, one needs to specify the site id and in the second column there has to be the name of the PFT.
+#'
+#' @examples
+loadPath.sitePFT<-function(settings, Path){
+  #finding the file extension.
+  ext <- tools::file_ext(Path)
+  
+  if (ext=="csv" | ext=="txt"){
+    # reading in the links
+    links <- read.table(file.path(settings$outdir,Path), header = T, sep = ",")
```

Comment:
I don't understand the assumption that the file will be located in the outdir. That seems pretty unlikely, as the outdir is run specific, while inputs are usually stored in a completely different directory and registered with the DB. Furthermore, our input processing code is set up to resolve the full paths of inputs. Seems like it would be simpler to just have this function take a single argument, Path.